### PR TITLE
Update Version To 2024.1.0.0

### PR DESCRIPTION
**Pull request by mryzhov**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -76,21 +85,20 @@ outputs:
...
       entry_points:
         - convert_tokenizer = openvino_tokenizers.cli:convert_hf_tokenizer
     requirements:
       build:
-        - python                              # [build_platform != target_platform]
-        - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+        - python >=3.8,<4
+        - pip
       host:
         - pip
```
Comment (ilya-lavrenov) [inline recipe/meta.yaml:99]:
```suggestion
```

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

**Comment by ilya-lavrenov** [comment]
Replaced by:
- https://github.com/conda-forge/openvino-tokenizers-feedstock/pull/12
- https://github.com/conda-forge/openvino-tokenizers-feedstock/pull/13

**Pull request closed without merging**